Add Documentation for .NET SDK Container Build (with Git LFS) (#6544)
microsoft/vscode-docs: docs/containers/debug-common.md

## .NET

> We offer two ways of building & debugging your project within Docker containers:

Reviewer: ```suggestion
You can choose between two ways of building and debugging your project within Docker containers:
```

Reviewer: Trying to avoid 'We' and spelling out the "and" symbol.

> - **With .NET SDK**: This option offers a simplified container debugging experience without the need for a `Dockerfile`. If you are familiar with `MSBuild` or want a straightforward approach, this is the recommended choice.

Author: here
